src/screens/UserPortal/Settings/Settings.spec.tsx

```
@@ -311,22 +176,39 @@ describe('Testing Settings Screen [User Portal]', () => {
-    expect(screen.getByTestId('inputGrade')).toHaveValue('GRADUATE');
-    expect(screen.getByTestId('inputEmpStatus')).toHaveValue('PART_TIME');
-    expect(screen.getByTestId('inputMaritalStatus')).toHaveValue('SINGLE');
-    expect(screen.getByTestId('inputAddress')).toHaveValue('random');
-    expect(screen.getByTestId('inputState')).toHaveValue('random');
-    expect(screen.getByTestId('inputCountry')).toHaveValue('IN');
-    expect(screen.getByLabelText('Birth Date')).toHaveValue('2024-03-01');
+    userEvent.type(screen.getByTestId('inputName'), 'Majumder');
+    await wait();
+
+    userEvent.click(screen.getByTestId('resetChangesBtn'));
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)